src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/toolbar_item.html.twig

```diff
@@ -1,6 +1,6 @@
 <div class="sf-toolbar-block sf-toolbar-block-{{ name }} sf-toolbar-status-{{ status|default('normal') }} {{ additional_classes|default('') }}">
-    {% if link %}<a href="{{ path('_profiler', { token: token, panel: name }) }}">{% endif %}
+    {% if link is defined and link %}<a href="{{ path('_profiler', { token: token, panel: name }) }}">{% endif %}
```

Comment:
In fact, the symfony core uses `profile_url` instead `true`, this can be confusing:
* [Bundle/WebProfilerBundle/Resources/views/Collector/twig.html.twig](https://github.com/symfony/symfony/blob/2.7/src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/twig.html.twig#L27)
* [Bundle/WebProfilerBundle/Resources/views/Collector/translation.html.twig](https://github.com/symfony/symfony/blob/2.7/src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/translation.html.twig#L37)
* [Bundle/WebProfilerBundle/Resources/views/Collector/time.html.twig]
(https://github.com/symfony/symfony/blob/2.7/src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/time.html.twig#L30)
* etc...